### ISSUE TITLE
Bug: Can't switch between gmail Accounts within Application

### ISSUE THREAD

**Issue by HollerAtTheCode** [ferdium/ferdium-app#880]
### Avoid duplicates

- [X] I have searched the [issues tracker](https://github.com/ferdium/ferdium-app/issues) for a bug report similar to mine, in vain

### Ferdium Version

6.2.3

### What Operating System are you using?

macOS

### Operating System Version

13.1 (22C65)

### What arch are you using?

arm64 (including Apple Silicon)

### Expected Behavior

Clicking the profile button in the gmail Service and selecting another account should change the inbox to the newly selected user user in ferdium.

### Actual Behavior

When switching the gmail account within ferdium the selected users inbox opens in the default web browser and not within ferdium.

### Steps to reproduce

1. click Profile Icon within ferdium gmail service.
2. Select another user profile.

### Additional information

Probably this issue should easily be fixed by applaying the setting to open links within ferdium to the gmail service.

**Comment by JPnux** [comment]
Yes, I still have the same problem with version 7.1. It's very annoying.
